docs/useful_tips.md

```diff
@@ -3,6 +3,8 @@
   - [Create a distrobox with a custom HOME directory](#create-a-distrobox-with-a-custom-home-directory)
   - [Mount additional volumes in a distrobox](#mount-additional-volumes-in-a-distrobox)
   - [Use a different shell than the host](#use-a-different-shell-than-the-host)
+  - [Run the container with real root](#run-the-container-with-real-root)
```

Comment:
What does this refer to?
`   - [Run the container with real root](#run-the-container-with-real-root)`

Reply:
It refers to this section (which was apparently missing in the list at the top):

https://github.com/89luca89/distrobox/blob/1e4d7b1cc914d05cb00ae8d7df5e5828c9cf43f6/docs/useful_tips.md?plain=1#L70-L83

Reply:
I would also add a clear warning that `root` inside a container that is running as `--root` or with `docker`, it's also root on the machine! It must be clear as it could be a dangerous thing